src/main/java/com/snowflake/kafka/connector/Utils.java

```diff
@@ -569,6 +573,7 @@ public static Map<String, String> parseTopicToTableMap(String input) {
       String topic = tt[0].trim();
       String table = tt[1].trim();
 
+      // TODO @rcheng: question - why the distinction between isInvalid and returning null?
```

Comment:
this function behaves differently when given different invalid topic to table maps. I think it should treat invalid table maps consistently - return null for the purposes of this PR. Can i make this change?

Reply:
sure. I think the general coding guideline is to not have multiple types of changes in one PR. so you can definitely make changes/refactor as required but may be add a TODO to do it in a separate PR?

Reply:
Sounds good, created a JIRA for it
https://snowflakecomputing.atlassian.net/browse/SNOW-659455